base_geoengine/fields.py

```diff
@@ -25,27 +28,140 @@ class GeoField(Field):
 
     geo_type = None
```

Comment:
I think that `geo_type` should become `column_type` and if it can be done it should remove some part of the code that alter table for geo type columns.